src/jiradeps/main.py

```diff
@@ -140,3 +140,7 @@ def log_issues(headline, issues):
     log.info('\n'.join([headline]
                        + ['    {} - {}'.format(issue.key, issue.fields.summary)
                           for issue in issues]))
+
+
+if __name__ == '__main__':
```

Comment:
Why do you put everything into a `src` directory but then want to directly run from the main module?

Reply:
Agreed, I'll remove it. At some point it was useful for testing.